Rlog/View/Components/UnderlinedTextField.swift

```diff
@@ -119,7 +123,7 @@ private struct UITextFieldRepresentable: UIViewRepresentable {
     }
     
     func updateUIView(_ uiView: UITextField, context: UIViewRepresentableContext<UITextFieldRepresentable>) {
-        uiView.text = self.text
+        uiView.text = String(describing: self.text)
```

Comment:
```suggestion
        uiView.text = "\(self.text)"
```

둘 다 동일합니다 그저 이런것도 있다 하는 공유예요 : )

Reply:
오오.. 감사합니다.
[407c994](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Bremen-DD/pull/33/commits/407c9946f17f09e77cd6d92af4aa7e98b15a176b) 로 반영하였습니다. suggestion을 선택하니 바로 반영이 되는군요..!